.github/workflows/cypress.yml

```
@@ -0,0 +1,37 @@
+name: E2E test
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+jobs:
+  cypress:
+    name: ${{ matrix.core.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        core:
+          - {name: 'WP latest', version: 'latest'}
+          - {name: 'WP minimum', version: 'WordPress/WordPress#5.7'}
```

```suggestion
          - {name: 'WP minimum', version: 'WordPress/WordPress#6.6'}
```

Per https://github.com/10up/block-catalog/blob/develop/block-catalog.php#L6C4-L6C26